src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -397,7 +397,7 @@ update_sc(unsigned int boardIdx, DSAInfo& candidate)
 
   // -- Some DRCs (Design Rule Checks) --
   // Is the SC present
-  if (current.bmc_ver().empty()) {
+  if (current.bmc_ver().empty() || candidate.bmc_ver().empty()) {
```

Comment:
This change addresses the SC CR. It is checking whether the shell package has SC available or not